src/utils/StorageHandler.ts

```diff
@@ -548,11 +548,17 @@ export default class StorageHandler {
       }
     } else {
       const allTargetApps = await this.getTargetAppNames();
+      const impactedTargetApps = new Set(entity.targetApps);
```

Comment:
How is `impactedTargetApps` used? It looks like it is only initialized and added to here.

Reply:
Good catch, missed the removal of this.